crates/agglayer-rpc/src/lib.rs

```diff
@@ -212,10 +217,14 @@ where
             latest_epoch_with_settlement: None,
         })
     }
+
+    pub fn get_current_epoch(&self) -> Option<agglayer_types::EpochNumber> {
```

Comment:
Is it really needed?

Reply:
Not really. Just wanted to use something from the epoch store to prevent unused warnings. I'll remove it.